classes/Entities/Cohort.php

```diff
@@ -20,4 +32,12 @@ public function courseCohortSyncs()
     {
         return $this->hasMany(CourseCohortSync::class,'cohortid','id');
     }
+
+    /**
+     * @return mixed
+     */
+    public function leaderboardsCohortSettings()
```

Comment:
Can you review the relationships that you've made and respective method names. belongsTo is singular, so the method name should signify that.

This should be leaderboardCohortSettings() for example.

Reply:
Done

Reply:
Should it not be _leaderboardCohortSetting()_ if there is just one or am i missing something?

Reply:
Done